app/ui-react/packages/models/src/models-internal.d.ts

```diff
@@ -17,6 +17,7 @@ export interface AcquisitionMethod {
   type?: 'OAUTH1' | 'OAUTH2';
```

Comment:
Wait, wasn’t this file in the gitignore?

Reply:
Gitignore doesnt move between repos

Reply:
I'll remove it, must be some magic git workflow I did...

Reply:
Seems that we [keep adding it](https://github.com/syndesisio/syndesis/commits/master/app/ui-react/packages/models/src/models-internal.d.ts), so it must be something with .gitignore